Code/Engine/Core/World/WorldModule.h

```diff
@@ -70,8 +70,8 @@ class EZ_CORE_DLL ezWorldModule : public ezReflectedClass
                                                   ///< called after all its dependencies have been called.
     ezEnum<ezWorldUpdatePhase> m_Phase;           ///< The update phase in which this update function should be called. See ezWorld for a description on the different phases.
     bool m_bOnlyUpdateWhenSimulating = false;     ///< The update function is only called when the world simulation is enabled.
-    ezUInt16 m_uiGranularity = 0;                 ///< The granularity in which batch updates should happen during the asynchronous phase. Has to be 0 for
-                                                  ///< synchronous functions.
+    ezUInt16 m_uiAsyncPhaseBatchSize = 0;         ///< 0 means all components are updated one after another, but still in parallel with other component managers.
```

Comment:
should in parallel with other world modules, they don't need to be component managers